{"name": "test_wrong_pwd", "status": "passed", "description": "密码输入6次错误", "attachments": [{"name": "log", "source": "2bde5d44-37e8-4812-bf3e-602b970ecaec-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "376d1064-d550-4e7e-8c90-f288b3b3b1aa-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "71391a89-f51b-4a2c-b77e-728ff78b3975-attachment.txt", "type": "text/plain"}], "start": 1632469574593, "stop": 1632469581765, "uuid": "ea538325-f940-4d4c-ad7d-cebfed19c167", "historyId": "7b9480ebb1961c7c14cb69cc7e65ba50", "testCaseId": "e4c5762512a76c681290f066bcc0e861", "fullName": "test_case.Login_case.TestLogin#test_wrong_pwd", "labels": [{"name": "severity", "value": "critical"}, {"name": "story", "value": "参数填写不正确，登录失败"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "Login_case"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KSOAST3"}, {"name": "thread", "value": "20116-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.Login_case"}]}